packages/mdc-menu/index.ts

```diff
@@ -21,192 +21,8 @@
-  }
-
-  set quickOpen(quickOpen: boolean) {
-    this.menuSurface_.quickOpen = quickOpen;
-  }
-
-  /**
-   * @param corner Default anchor corner alignment of top-left menu corner.
-   */
-  setAnchorCorner(corner: Corner) {
-    this.menuSurface_.setAnchorCorner(corner);
-  }
-
-  setAnchorMargin(margin: Partial<MenuDistance>) {
-    this.menuSurface_.setAnchorMargin(margin);
-  }
-
-  /**
-   * @return The item within the menu at the index specified.
-   */
-  getOptionByIndex(index: number): Element | null {
-    const items = this.items;
-
-    if (index < items.length) {
-      return this.items[index];
-    } else {
-      return null;
-    }
-  }
-
-  setFixedPosition(isFixed: boolean) {
-    this.menuSurface_.setFixedPosition(isFixed);
-  }
-
-  hoistMenuToBody() {
-    this.menuSurface_.hoistMenuToBody();
-  }
-
-  setIsHoisted(isHoisted: boolean) {
-    this.menuSurface_.setIsHoisted(isHoisted);
-  }
-
-  setAbsolutePosition(x: number, y: number) {
-    this.menuSurface_.setAbsolutePosition(x, y);
-  }
-
-  /**
-   * Sets the element that the menu-surface is anchored to.
-   */
-  setAnchorElement(element: Element) {
-    this.menuSurface_.anchorElement = element;
-  }
-
-  handleAfterOpened_() {
-    const list = this.items;
-    if (list.length > 0) {
-      (list[0] as HTMLElement).focus();
-    }
-  }
-
-  getDefaultFoundation(): MDCMenuFoundation {
-    // tslint:disable:object-literal-sort-keys
-    return new MDCMenuFoundation({
-      addClassToElementAtIndex: (index, className) => {
-        const list = this.items;
-        list[index].classList.add(className);
-      },
-      removeClassFromElementAtIndex: (index, className) => {
-        const list = this.items;
-        list[index].classList.remove(className);
-      },
-      addAttributeToElementAtIndex: (index, attr, value) => {
-        const list = this.items;
-        list[index].setAttribute(attr, value);
-      },
-      removeAttributeFromElementAtIndex: (index, attr) => {
-        const list = this.items;
-        list[index].removeAttribute(attr);
-      },
-      elementContainsClass: (element, className) => element.classList.contains(className),
-      closeSurface: () => this.open = false,
-      getElementIndex: (element) => this.items.indexOf(element),
-      getParentElement: (element) => element.parentElement,
-      getSelectedElementIndex: (selectionGroup) => {
-        const selectedListItem = selectionGroup.querySelector(`.${cssClasses.MENU_SELECTED_LIST_ITEM}`);
-        return selectedListItem ? this.items.indexOf(selectedListItem) : -1;
-      },
-      notifySelected: (evtData) => this.emit<DefaultMenuItemEventDetail>(strings.SELECTED_EVENT, {
-        index: evtData.index,
-        item: this.items[evtData.index],
-      }),
-    });
-    // tslint:enable:object-literal-sort-keys
-  }
-}
-
-export {MDCMenu as default, MDCMenu, Corner};
+export {Corner} from '@material/menu-surface/constants'; // for backward compatibility
```

Comment:
I guess AnchorMargin no longer exists, but again it's just typings so probably not critical to lose it here.

Reply:
Yep, this shouldn't break anybody. (`AnchorMargin` was renamed to `MDCMenuDistance`.)